applications/callflow/src/module/cf_park.erl

```diff
@@ -643,10 +643,14 @@ unanswered_action(SlotNumber, Slot, Data, Call) ->
 
 -spec presence_type(ne_binary(), wh_json:object(), whapps_call:call()) -> ne_binary().
 presence_type(SlotNumber, Data, Call) ->
-    case wh_json:get_value(?PRESENCE_TYPE_KEY, slot_configuration(Data, SlotNumber)) of
-        'undefined' -> ?ACCOUNT_PARKED_TYPE(whapps_call:account_id(Call));
-        Type -> Type
-    end.
+    JObj = slot_configuration(Data, SlotNumber),
+    DefaultPresenceType =
+        wh_json:get_ne_value(
```

Comment:
@k-anderson why `wh_json:get_ne_value` and not `wh_json:get_ne_binary_value` to follow the spec ?